.github/workflows/connect-dependencies-npm-deploy.yml

```diff
@@ -0,0 +1,45 @@
+name: "[Release] connect dependencies npm"
+on:
+  workflow_call:
+    inputs:
+      package:
+        description: Name of the package to deploy.
+        required: true
+        type: choice
+        options:
+          - blockchain-link-types
+          - blockchain-link-utils
+          - blockchain-link
+          - connect-common
+          - transport
+          - utils
+          - utxo-lib
+          - connect-plugin-stellar
+          - connect-plugin-ethereum
+          - analytics
+          - connect-analytics
+          - type-utils
+          - env-utils
+          - protocol
+          - protobuf
+          - schema-utils
+
+jobs:
+  extract-branch:
```

Comment:
why is this a separate job? this shouldn't be neededed as standalone job, but rather a step in the job we actually need.

Reply:
Right, it is in the job now, I removed it as separate job and as input for template.